public/scripts/wiki.js

```diff
@@ -111,6 +114,92 @@ var ArticleBox = React.createClass({
 
 });
 
+var Navbar = React.createClass({
+  getInitialState: function() {
+    return {user: '',loginshow:'',logoutshow:'none'};
+  },
+  loginFacebook: function(){
+    $.ajax({
+      url: '/login',
+      dataType: 'json',
+      type: 'GET',
+      success: function(data) {
+        this.setState({user: data, loginshow:'none', logoutshow:''});
+        console.log(data)
+      }.bind(this),
+      error: function(xhr, status, err) {
+        this.setState({user: '',loginshow:'',logoutshow:'none'});
+        // console.error(this.props.url, status, err.toString());
+      }.bind(this)
+    });
+  },
+  componentDidMount: function() {
+    this.loginFacebook();
+  },
+  render: function(){
+    var loginshow = this.state.loginshow;
+    var logoutshow = this.state.logoutshow;
+    // console.log(req.session.passport);
+    return (
+      <div className="Navbar navbar navbar-default">
+        <a className="navbar-brand"> Movie Wiki </a>
+        <div className="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+          <ul className="nav navbar-nav navbar-right">
+            <li><a style={{color:"#148a66"}}>{this.state.user.username}</a></li>
+            <li><a href="/auth/facebook" style={{display:loginshow}}><i className="fa fa-facebook">Login</i></a></li>
+            <li><a href="/logout" style={{display:logoutshow}}><i className="fa fa-facebook">Logout</i></a></li>
+          </ul>
+          <SearchForm onArticleSearch={this.props.onArticleSearch}/>
+        </div>
+      </div>
+      );
+  }
+});
+
+var SearchForm = React.createClass({
```

Comment:
cool, I didn't realize we were adding a search form but this looks great! i'll go test it :tulip: